ydb/core/cms/console/console_configs_manager.cpp

```diff
@@ -67,6 +68,73 @@ bool TConfigsManager::CheckConfig(const NKikimrConsole::TConfigsConfig &config,
     return true;
 }
 
+TConfigsManager::TValidateConfigResult TConfigsManager::ValidateYamlConfig(const TString &config, bool force, bool allowUnknownFields) {
```

Comment:
Method itself seem's fine, but I'm not sure about naming it's not just validation, it's also some processing (replace metadata). So it's better to rename this function, or extract ReplaceMetadata call